src/3box.js

```diff
@@ -211,6 +250,21 @@ class Box {
     }
   }
 
+  /**
+   * GraphQL for 3Box profile API
+   *
+   * @param     {Object}    query               A graphQL query object.
+   * @param     {Object}    opts                Optional parameters
+   * @param     {String}    opts.graphql        URL of graphQL 3Box profile service
```

Comment:
Maybe something like `opts.serviceUrl` is more descriptive?

Reply:
Maybe `graphqlServer` to be consistent.

Reply:
yeah graphqlServer is good